src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -7,12 +7,13 @@
 }
 
 .worklist-list__item-check {
-  cursor: default;
   padding: 16px 0;
   width: 16px;
+}
 
-  .fa-check-square {
-    color: color(green);
+.work-list__item:hover {
+  .fa-square {
+    color: $black-20;
```

Comment:
why is this needed and how is it work-list specific?

Reply:
Mark asked that the checkbox have the hover state when a user hovers anywhere on the row.

Reply:
🙄